runtime/hydradx/src/assets.rs

```diff
@@ -165,19 +334,24 @@ impl pallet_claims::Config for Runtime {
 }
 
 parameter_types! {
+	#[derive(PartialEq, Debug)]
```

Comment:
why this is needed here ? u32 should already support these traits

Reply:
Rust started to complain and I had to add these when I removed `BoundedVec` from `AssetDetails`

Reply:
isnt this caused by the fact that in the config for the StringLimit you explicity added these two traits ? which i dont think it should be needed.

Reply:
yes, I added these because without it I wasn't able to assert in tests and benchmarks. If I remove it, I still need add these derives because of benchmakrs